src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,21 @@
-export const TodoList = () => {};
+import { CompletedTodo } from '../../type';
+import { TodoInfo } from '../TodoInfo';
+
+type Prop = {
+  todos: CompletedTodo[]
+};
+
+export const TodoList: React.FC<Prop> = ({ todos }) => {
+  return (
+    <section className="TodoList">
+      {todos.map((completedTodo: CompletedTodo) => {
+        return (
+          <TodoInfo
+            key={completedTodo.todo.id}
+            {...completedTodo}
```

Comment:
better pass `todo` here as props. But with destructuring it would look a little different.
```suggestion
            todo={completedTodo.todo}
```